lib/dpl/provider/gae.rb

```diff
@@ -72,7 +72,7 @@ def push_app
         command << " --verbosity \"#{verbosity}\""
         command << " --project \"#{project}\""
         command << " app deploy \"#{config}\""
-        command << " --version \"#{version}\""
+        command << (version ? " --version \"#{version}\"" : '')
```

Comment:
I don't think this works if `version` was explicitly given as `""`, as unlikely as it might be. (This actually is relevant to the `no_stop_previous_version` option, now that I think about it.)

A more robust way for this is:

```ruby
command << " --version \"#{version}\"" unless version.empty?
```

(And similar for `no_stop_previous_version`.)